toutvcli/app.py

```diff
@@ -31,7 +31,6 @@
 import distutils.version
```

Comment:
It seems like I can't leave a message on the commit log... but could you provide a before/after example in the log?